services/wercker/wercker.service.js

```diff
@@ -23,7 +23,7 @@ module.exports = class Wercker extends BaseJsonService {
     return {
       base: 'wercker',
       format:
-        '(?:(?:ci/)([a-fA-F0-9]{24})|(?:build|ci)/([^/]+/[^/]+))(?:/(.+))?',
+        '(?:(?:ci/)([a-fA-F0-9]{24})|(?:build|ci)/([^/]+/[^/]+?))(?:/(.+?))?',
```

Comment:
I'm getting a `404 | badge not found` with the wercker ci badge (no branch) in the review app. Not sure what the cause is

https://shields-staging-pr-3717.herokuapp.com/wercker/ci/59e33c8e982fc615500b357

![](https://shields-staging-pr-3717.herokuapp.com/wercker/ci/59e33c8e982fc615500b357)

Reply:
This is 404 too:

https://img.shields.io/wercker/ci/59e33c8e982fc615500b357.svg

Reply:
well that's a bummer for the wercker badges, but at least not related to this PR 👍

Reply:
its because `59e33c8e982fc615500b357` is only 23 characters long. https://img.shields.io/wercker/ci/559e33c8e982fc615500b357.svg does work

Reply:
Ha! User error on my part not being able to correctly copy and paste 🤦‍♂ 

Good spot